buildSrc/src/main/java/quilt/internal/tasks/diff/CheckTargetVersionExistsTask.java

```diff
@@ -9,50 +9,54 @@
 
 import com.google.gson.JsonElement;
 import com.google.gson.JsonParser;
+import org.gradle.api.file.RegularFileProperty;
 import org.gradle.api.tasks.Internal;
 import org.gradle.api.tasks.OutputFile;
 import org.gradle.api.tasks.TaskAction;
 import quilt.internal.Constants;
 import quilt.internal.tasks.DefaultMappingsTask;
 
-public class CheckTargetVersionExistsTask extends DefaultMappingsTask {
+public abstract class CheckTargetVersionExistsTask extends DefaultMappingsTask {
     public static final String TASK_NAME = "checkTargetVersionExists";
 
     @Internal
```

Comment:
Uh... I don't like whatever this is doing. Does it set the field _during task execution_ then expect tasks running after it to grab the field from the task and use it? Iffy, to say the least, though I suppose functional. Regardless -- I'd annotate the method, not the field, here in either case.